scala/akkahttp/project/assembly.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.4")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
```

Comment:
could we use the last version (if it is safe) ?

Reply:
0.14.10 is the latest as far as I can see